wandb/run_manager.py

```diff
@@ -983,7 +983,7 @@ def _upsert_run(self, retry, storage_id, env):
                     wandb.termerror(
                         'Failed to connect to W&B. Retrying in the background.')
                     return False
-                launch_error_s = 'Launch exception: {}, see {} for details.  To disable wandb set WANDB_MODE=dryrun'.format(e, util.get_log_file_path())
```

Comment:
why take out the extra context?

Reply:
We already display a link to the debug log, it was redundant.